### PR TITLE
Bump Ingest SDK to 1.1.0 and release 1.9.0

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
This will not work unless we releases 1.1.0.
I will wait for it and merge gates to pass, I am just keeping PR ready.

### CODE REVIEW

**Review by sfc-gh-tjones — APPROVED** [review]

**Comment by sfc-gh-xhuang** [comment]
Do we want this PR merged too?
https://github.com/snowflakedb/snowflake-kafka-connector/pull/551

**Comment by sfc-gh-japatel** [comment]
> Do we want this PR merged too? #551

That PR is not yet complete, I dont see this as a blocker/must have for 1.9.0. Once @sfc-gh-rcheng is back, she can continue to work on this and we can make it available in later versions.

File: test/test_suit/test_schema_mapping.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(self, driver, nameSalt):
         self.gold = {
             'PERFORMANCE_STRING': 'Excellent',
             'case_sensitive_PERFORMANCE_CHAR': 'A',
-            'PERFORMANCE_HEX': b'\xef\xbf\xbd\xef\xbf\xbd\xef\xbf\xbd\xef\xbf\xbd',
+            'PERFORMANCE_HEX': b'\xff\xff\xff\xff',
```
Comment (sfc-gh-tzhang) [inline test/test_suit/test_schema_mapping.py:39]: @sfc-gh-lsembera any idea which change from 1.0.3-beta to 1.1.0 cause this behavior change? The new value looks right to me though.....

File: test/test_suit/test_schema_mapping.py [hunk 2]
```diff
@@ -88,9 +88,14 @@ def verify(self, round):
...
                 continue;
             if type(res[res_col[field]]) == str:
+                print(res[res_col[field]])
+                print(self.gold[field])
                 # removing the formating created by sf
                 assert ''.join(res[res_col[field]].split()) == self.gold[field]
             else:
+                print(type(res[res_col[field]]))
+                print(res[res_col[field]])
+                print(self.gold[field])
```
Comment (sfc-gh-tzhang) [inline test/test_suit/test_schema_mapping.py:98]: nit, we can just print the value if the assert fails?

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM, let's go!

File: test/test_suit/test_schema_mapping.py (hunk 2 above)
Comment (sfc-gh-japatel) [reply test/test_suit/test_schema_mapping.py:98]: yes I will remove this..

**Pull request merged**